components/modules/node.c

```diff
@@ -106,14 +106,17 @@ static int node_bootreason( lua_State *L)
     case EXT_CPU_RESET:
 #endif
     case DEEPSLEEP_RESET:
-#if defined(CONFIG_IDF_TARGET_ESP32) || defined(CONFIG_IDF_TARGET_ESP32C3)
+#if defined(CONFIG_IDF_TARGET_ESP32)
     case SDIO_RESET:
 #endif
-#if defined(CONFIG_IDF_TARGET_ESP32S2) || defined(CONFIG_IDF_TARGET_ESP32S3)
+#if defined(CONFIG_IDF_TARGET_ESP32S2) || defined(CONFIG_IDF_TARGET_ESP32S3) || defined(CONFIG_IDF_TARGET_ESP32C3)
     case GLITCH_RTC_RESET:
 #endif
-#if defined(CONFIG_IDF_TARGET_ESP32S3)
+#if defined(CONFIG_IDF_TARGET_ESP32S3) || defined(CONFIG_IDF_TARGET_ESP32C3)
     case EFUSE_RESET:
+    case USB_UART_CHIP_RESET:
+    case USB_JTAG_CHIP_RESET:
+    case POWER_GLITCH_RESET:
```

Comment:
These look like non-RMT related fixes, should they go in a separate PR perhaps?

Reply:
I think that this should have gone in the idf4.4 update PR.